<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KbLayoutSetupWidget</class>
 <widget class="QWidget" name="KbLayoutSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="./libpcbsd-ui.qrc">
         <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="./libpcbsd-ui.qrc">
         <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="./libpcbsd-ui.qrc">
         <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="upBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="./libpcbsd-ui.qrc">
         <normaloff>:/images/arrow-up.png</normaloff>:/images/arrow-up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="downBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="./libpcbsd-ui.qrc">
         <normaloff>:/images/arrow-down.png</normaloff>:/images/arrow-down.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="layoutsTW">
     <column>
      <property name="text">
       <string>-</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Input language</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Variant</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="./libpcbsd-ui.qrc"/>
 </resources>
 <connections/>
</ui>
